src/components/CodeViewer/CodeTabs.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./CodeTabs.css";
+
+interface CodeTabProps {
+    title: string;
+    language: string;
+    children: React.ReactNode;
+}
+
+export default function CodeTab({ children }: CodeTabProps): JSX.Element {
+    return <>{children}</>;
```

Comment:
What's this?

Reply:
Types for the CodeViewer input.
The CodeViewer don't require this type as childrends because I don't know how to type it (non simple knowledge of react types is needed).
But this component is currently used when passing childrens to the CodeViewer, so we got a bit of type safety.